revised/beamswap-proposal.md

```diff
@@ -0,0 +1,330 @@
+
+### Phase 2
+
+Following the successful Phase 1 of building sufficient liquidity, the Beamex perpetual DEX, the first perpetual exchange on Polkadot Network, is planned to launch.
+
+Using Beamswap Flywheel, users will be able to benefit from Moonbeam and Polkadot capabilities to open long or short positions on Beamex-supported assets.
+
+![|602x339](upload://4UU32xt3zIprYMzEIq4fgiebraE.jpeg)
+
+Expected APY and TVL should be met within the second week of the Beamex launch. The adoption of Beamex will be supported through a set of guides, dedicated community managers, and daily marketing activity to ease user onboarding to a completely new product. The use of Beamex Beta is strongly encouraged.
+
+## Roll out of BLP token powering Beamex liquidity
+
+### BLP token
+
+BLP, Beamex Liquidity Provider Index Token, is Beamex’s native liquidity provider asset.
+
+It represents an index of assets used to support swaps and leverage trading on Beamex, and replaces the liquidity token pairs used to provide liquidity on Beamswap.
+
+BLP is composed of:
+
+* ETH (30%)
+* BTC (12%)
+* DOT (5%)
+* GLMR (3%)
+* USDC (50%)
+
+BLP can be minted using any index asset or burned to redeem any index asset. In case the situation with USDC and Circle deteriorates, the stablecoin would be replaced with USDT (Tether) prior to product launch.
+
+Leverage traders source liquidity by borrowing from exchange, or, more accurately, liquidity providers or BLP token holders. When such traders suffer a loss, BLP holders earn a profit and vice versa.
+
+### Phase 1
+
+The initial phase is the Launch phase with 140,000 GLMR community rewards.
+
+### Phase 2
+
+The following phase is a Retail phase with 310,000 GLMR community rewards.
+
+![|602x339](upload://jah9xRmlIOoddRGwE5EnuJ09hdO.jpeg)
+
+# 10 Vision of Success
+
+This proposal aims not to simply funnel granted funds to increase the platform’s volume and liquidity. Instead, it seeks to spread them more thoroughly and thoughtfully to primarily cater to user expectations and demands to enjoy an excellent DeFi experience.
+
+While the number of users is easy to count, it is much more challenging to predict the user base growth rate. User adoption depends not only on the success and functionalities of the platform but also on factors beyond the team’s control, such as network stability, market sentiment, competition, media presence, and word of mouth.
+
+At the beginning of 2023, Beamswap had an average of 2,661 unique users per month and added 953 new users per month. These figures will be used as a baseline for determining the Incentive Program’s success along with TVL, volume, and transaction count.
+
+As Beamswap progresses with platform development, continuously steady QOQ user growth is anticipated, especially with incentives powering the user onboarding initiatives and campaigns.
+
+![|602x339](upload://91jHDEqQmYsNh2v469wzV5IF4Ac.jpeg)
```

Comment:
Can you also upload the images to the revised/assets/beamswap folder? Otherwise we'll end up with some empty placeholders.